tests/createSockDivTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class createSockDivTest extends TestCase {
+    public function testCreateSockDiv(): void
```

Comment:
what kind of test is this? success? fail? malformed?

Reply:
should I add the type into the test name eg. creatSockDivSuccessTest?

Reply:
yes, `createSockDivTest_Success` would be standard